crates/bevy_window/src/lib.rs

```diff
@@ -56,6 +56,7 @@ impl Plugin for WindowPlugin {
                 .get_resource::<WindowDescriptor>()
```

Comment:
I think remove_resource also returns the resource, right?

That is, these two lines could be merged.